### DISCUSSION TITLE
Docs: How to use redux with app router correctly

### DISCUSSION THREAD

**Discussion by billymoon** [vercel/next.js#68556] [Help]
### What is the update you wish to see?

When I follow the guidance from the [redux website to update the per-route state](https://redux.js.org/usage/nextjs#per-route-state), I encounter an error. This error leads to very slow navigation in my app. I would like guidance on how to correctly implement redux in a nextjs app with the app router.

### Is there any context that might help us understand?

I have created a minimal reproduction of the issue based off the with-redux example

https://github.com/vercel/next.js/commit/7343c29389d189421b6c27a4d3b5617f75a3062a

explained here with screenshots...

https://github.com/billymoon/next.js/blob/canary/examples/with-redux/problem.md

**Comment by samcx** [comment]
@billymoon Converting this to our [Discussions](https://github.com/vercel/next.js/discussions) as it is more of a request than an issue with our Documentation (e.g., incorrect information).
